<launch>
    <node
        type="turtlesim_node"
        name="turtlesim_node"
        pkg="turtlesim" 
        >
    <remap
        from="turtle1/cmd_vel"
        to="turtle1/cmd_vel_reversed"
        />
    </node>
    <node
        pkg="turtlesim"
        type="turtle_teleop_key"
        name="teleop_key"
        launch-prefix="xterm -e"
    />
    <node
        pkg="agitr"
        type="reverse_cmd_vel"
        name="reverse_velocity"
        />
</launch>